lib/galaxy/tool_util/linters/help.py

```diff
@@ -30,10 +30,22 @@ def lint_help(tool_xml, lint_ctx):
     if "TODO" in help_text:
         lint_ctx.warn("Help contains TODO text.", node=helps[0])
 
+    invalid_rst = rst_invalid(help_text)
+    if invalid_rst:
+        lint_ctx.warn(f"Invalid reStructuredText found in help - [{invalid_rst}].", node=helps[0])
+    else:
+        lint_ctx.valid("Help contains valid reStructuredText.", node=helps[0])
+
+
+def rst_invalid(text):
+    """
+    Predicate to determine if text is invalid reStructuredText.
+    Return False if the supplied text is valid reStructuredText or
+    a string indicating the problem.
+    """
+    invalid_rst = False
```

Comment:
```suggestion
    invalid_rst: Union[bool, str] = False
```
and import Union at the top